FourthTask/figures/types/types.go

```diff
@@ -0,0 +1,70 @@
+//	import $GOPATH/src/github.com/Mishamba/EPAM_GOLANG_COURSE/FourthTask/figures/types
+//	Package contains structures descriptions and constructors, interface definition
+
+package types
+
+import (
+	"errors"
+	"math"
+)
+
+const (
+	circlePanic   string = " circle can't has negative radius"
+	squarePanic   string = " square can't has negative side"
+	overflowError string = " result will be bigger, that we can coutn. sorry..."
+)
+
+type Figure interface {
```

Comment:
Seems you should Export funcs `Area()` and `Perimeter()`  in this interface.
Also, a minor, it's better to keep english names, so `Perimeter` would be better here.